less/home-page.less

```diff
@@ -1,22 +1,46 @@
 .home {
 
+  header {
```

Comment:
I know you can see what's going on here by reading the code, but I recommend adding comments above each section so that nested sections can be found at a glance. For examples, // Header Styles above this section.